README.md

```diff
@@ -724,6 +724,11 @@ scanning in the background. Disable antivirus scanner and try again.
 [multirust]: https://github.com/brson/multirust
 [Diggory Blake]: https://github.com/Diggsey
 
+### I get "error: could not remove 'rustup-bin' file: 'C:\Users\USER\.cargo\bin\rustup.exe'"
```

Comment:
Might need `\\.cargo` as markdown escape it somehow.

Reply:
Looking at the render, it looks okay.

Reply:
adding the extra \